setuptools/tests/fixtures.py

```diff
@@ -98,7 +98,18 @@ def venv(tmp_path, setuptools_wheel):
     env = environment.VirtualEnv()
     env.root = path.Path(tmp_path / 'venv')
     env.req = str(setuptools_wheel)
-    return env.create()
+    # In some environments (eg. downstream distro packaging),
+    # where tox isn't used to run tests and PYTHONPATH is set to point to
+    # a specific setuptools codebase, that PYTHONPATH leaks to the spawned
```

Comment:
Thank you very much @befeleme, that is a very great summary. 

```suggestion
    # a specific setuptools codebase, PYTHONPATH will leak into the spawned
```

I think it read better like this.

Reply:
I will just go ahead and commit the suggestions, this way the CI can run all at once.